.VERSION

```diff
@@ -1,6 +1,6 @@
 $Format:%d%n%n$
 # Fall back version, probably last release:
-2.3.1
+2.5.0
```

Comment:
This is a duplicate change with your other PR. If you were to have based this branch from master I can do an octopus merge later and merge both at once.

Reply:
Oops.  I just merged the other one and deleted the branch.  I assume there's no problem with merging this one too.  I would expect `git` is smart enough to recognize the duplicate commit and only add in what's new.  I hope it's ok if I go for it.